<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlastDatabaseDownloadWindow</class>
 <widget class="QWidget" name="BlastDatabaseDownloadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Download BLAST database</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/resources/images/icons/seanau/Database share.png</normaloff>:/resources/images/icons/seanau/Database share.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="verticalSpacing">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Select BLAST database:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTableWidget" name="databaseTableWidget">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>22</number>
           </attribute>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>5</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>6</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>7</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>8</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>9</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>10</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>11</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>12</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>13</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>14</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>15</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>16</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>17</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>18</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>19</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>20</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>21</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>22</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>23</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Description</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>env_nr</string>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>Environmental protein sequences</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>env_nt</string>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string>Environmental nucleotide sequences
</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>est</string>
            </property>
           </item>
           <item row="2" column="1">
            <property name="text">
             <string>Est database from the EST division of GenBank, EMBL, and DDBJ</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>est_human</string>
            </property>
           </item>
           <item row="3" column="1">
            <property name="text">
             <string>Human subset of the est</string>
            </property>
           </item>
           <item row="4" column="0">
            <property name="text">
             <string>est_mouse</string>
            </property>
           </item>
           <item row="4" column="1">
            <property name="text">
             <string>Mouse subset of the est</string>
            </property>
           </item>
           <item row="5" column="0">
            <property name="text">
             <string>est_others</string>
            </property>
           </item>
           <item row="5" column="1">
            <property name="text">
             <string>Non-human and non-mouse subset of est</string>
            </property>
           </item>
           <item row="6" column="0">
            <property name="text">
             <string>gss</string>
            </property>
           </item>
           <item row="6" column="1">
            <property name="text">
             <string>GSS division of GenBank, EMBL, and DDBJ</string>
            </property>
           </item>
           <item row="7" column="0">
            <property name="text">
             <string>htgs</string>
            </property>
           </item>
           <item row="7" column="1">
            <property name="text">
             <string>HTG division of GenBank, EMBL, and DDBJ</string>
            </property>
           </item>
           <item row="8" column="0">
            <property name="text">
             <string>human_genomic</string>
            </property>
           </item>
           <item row="8" column="1">
            <property name="text">
             <string>Human RefSeq (NC_######) chromosome records with gap adjusted concatenated NT_ contigs</string>
            </property>
           </item>
           <item row="9" column="0">
            <property name="text">
             <string>nr</string>
            </property>
           </item>
           <item row="9" column="1">
            <property name="text">
             <string>Non-redundant protein sequence database with entries from GenPept, SwissProt, PIR, PDF, PDB, and NCBI RefSeq</string>
            </property>
           </item>
           <item row="10" column="0">
            <property name="text">
             <string>nt</string>
            </property>
           </item>
           <item row="10" column="1">
            <property name="text">
             <string>Nucleotide sequence database with entries from all traditional divisions of GenBank, EMBL, and DDBJ excluding bulk divisions (gss, sts, pat, est, htg, wgs). Not non-redundant</string>
            </property>
           </item>
           <item row="11" column="0">
            <property name="text">
             <string>other_genomic</string>
            </property>
           </item>
           <item row="11" column="1">
            <property name="text">
             <string>RefSeq chromosome records (NC_######) for organisms other than human</string>
            </property>
           </item>
           <item row="12" column="0">
            <property name="text">
             <string>pataa</string>
            </property>
           </item>
           <item row="12" column="1">
            <property name="text">
             <string>Patent protein sequence database</string>
            </property>
           </item>
           <item row="13" column="0">
            <property name="text">
             <string>patnt</string>
            </property>
           </item>
           <item row="13" column="1">
            <property name="text">
             <string>Patent nucleotide sequence database</string>
            </property>
           </item>
           <item row="14" column="0">
            <property name="text">
             <string>pdbaa</string>
            </property>
           </item>
           <item row="14" column="1">
            <property name="text">
             <string>Protein sequences from PDB protein structures</string>
            </property>
           </item>
           <item row="15" column="0">
            <property name="text">
             <string>pdbnt</string>
            </property>
           </item>
           <item row="15" column="1">
            <property name="text">
             <string>Nucleotide sequences from pdb nucleic acid structures</string>
            </property>
           </item>
           <item row="16" column="0">
            <property name="text">
             <string>refseq_genomic</string>
            </property>
           </item>
           <item row="16" column="1">
            <property name="text">
             <string>NCBI genomic reference sequences</string>
            </property>
           </item>
           <item row="17" column="0">
            <property name="text">
             <string>refseq_protein</string>
            </property>
           </item>
           <item row="17" column="1">
            <property name="text">
             <string>NCBI protein reference sequences</string>
            </property>
           </item>
           <item row="18" column="0">
            <property name="text">
             <string>refseq_rna</string>
            </property>
           </item>
           <item row="18" column="1">
            <property name="text">
             <string>NCBI transcript reference sequences</string>
            </property>
           </item>
           <item row="19" column="0">
            <property name="text">
             <string>sts</string>
            </property>
           </item>
           <item row="19" column="1">
            <property name="text">
             <string>Sequences from the STS division of GenBank, EMBL, and DDBJ</string>
            </property>
           </item>
           <item row="20" column="0">
            <property name="text">
             <string>swissprot</string>
            </property>
           </item>
           <item row="20" column="1">
            <property name="text">
             <string>Swiss-Prot sequence database (last major update)</string>
            </property>
           </item>
           <item row="21" column="0">
            <property name="text">
             <string>taxdb</string>
            </property>
           </item>
           <item row="21" column="1">
            <property name="text">
             <string>Additional taxonomic information for the formatted database (contains common and scientific names)</string>
            </property>
           </item>
           <item row="22" column="0">
            <property name="text">
             <string>wgs</string>
            </property>
           </item>
           <item row="22" column="1">
            <property name="text">
             <string>Whole genome shotgun sequence assemblies</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Database description:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="databaseDescriptionLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Database description label</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Destination:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="destinationComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>&amp;Cancel</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources.qrc">
             <normaloff>:/resources/images/icons/seanau/Cancel.png</normaloff>:/resources/images/icons/seanau/Cancel.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Start download</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources.qrc">
             <normaloff>:/aliases/images/icons/arrow-right</normaloff>:/aliases/images/icons/arrow-right</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Downloading:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLabel" name="activeDbLabel">
           <property name="text">
            <string>Download label</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Destination:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLabel" name="activeDestinationLabel">
           <property name="text">
            <string>Destination</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Time elapsed:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLabel" name="timeElapsedLabel">
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QLabel" name="actionTextLabel">
           <property name="text">
            <string>Action text!</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QProgressBar" name="downloadProgressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="0" rowspan="2">
          <widget class="QPushButton" name="cancelDownloadButton">
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources.qrc">
             <normaloff>:/resources/images/icons/seanau/Cancel.png</normaloff>:/resources/images/icons/seanau/Cancel.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>170</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="successButton">
           <property name="text">
            <string>Success</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources.qrc">
             <normaloff>:/resources/images/icons/seanau/Success.png</normaloff>:/resources/images/icons/seanau/Success.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>239</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>released()</signal>
   <receiver>BlastDatabaseDownloadWindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
